mk/spksrc.download.mk

```diff
@@ -60,10 +60,11 @@ download_target: $(PRE_DOWNLOAD_TARGET)
 	    git) \
 	      localFolder=$(NAME)-git$(PKG_GIT_HASH) ; \
 	      localFile=$${localFolder}.tar.gz ; \
+	      for i in {1..12}; do [ -f $${localFolder}.part ] && sleep 10 ; done ; \
```

Comment:
A good idea. I would set a constant to avoid this "12" magic number three times in file... In case someone would be interested to increase it...

Reply:
totally agree, now done.  Also did the same for `--timeout` flock param.

Reply:
Also added a `|| break` to not be spending wasted cycles in the for look.  The one-line is now:
```
for i in {1..$${WAIT_MAX}}; do [ -f $${localFile}.part ] && sleep 10 || break; done ; \
```